src/components/HeartbeatBar.vue

```diff
@@ -111,6 +98,47 @@ export default {
             return placeholders.concat(this.beatList.slice(start));
         },
 
+        values() {
+            if (!this.shortBeatList || !this.shortBeatList.length) {
+                return [];
+            }
+
+            const valueObj = {};
+
+            this.shortBeatList.forEach(({ status, time }) => {
+                const date = dayjs(time).format("YYYY-MM-DD");
+
+                if (valueObj[date] === undefined) {
+                    valueObj[date] = 0;
+                }
+
+                let count = 0;
+
+                switch (status) {
+                    case 0:
+                        break;
+                    case 1:
+                        count = 10;
+                        break;
+                    case 2:
+                        count = 2;
+                        break;
+                    case 3:
+                        count = 5;
+                }
+
+                valueObj[date] += count;
+            });
+
+            return Object.keys(valueObj).map(date => ({ date,
+                count: valueObj[date] }));
+        },
+
+        endDate() {
+            const date = dayjs().format("YYYY-MM-DD");
+            return date;
```

Comment:
nit:I think it might be worth considering to inline this function above given that this is just a method call into dayjs

```suggestion
            return dayjs().format("YYYY-MM-DD");
```